plugins/go/README.md

```diff
@@ -0,0 +1,34 @@
+# Go plugin for MXE
+
+See also article [cross-compile go code, including cgo][1]
+by Dimitri John Ledkov.
+
+[1]: http://blog.surgut.co.uk/2014/06/cross-compile-go-code-including-cgo.html
+
+Package `go-native` installs native Go 1.4. This version of Go
+doesn't depend on Go installation.
+
+Package `go` uses native Go 1.4 as a bootstrap and installs Go 1.6
+as a cross-compiler to windows/386 or windows/amd64. Versions of
+Go starting with 1.5 need Go installation to build.
+
+To build Go packages for windows/386 or windows/amd64, you have to set
+the [GOPATH](https://golang.org/doc/code.html#GOPATH) environment variable
+and call `usr/bin/$(TARGET)-go` wrapper script.
+
+## Example
+
+Building [gohs](https://github.com/flier/gohs), GoLang Binding of
+[HyperScan](https://01.org/hyperscan).
+
+```
+$ MXE_PLUGIN_DIRS=plugins/go make hyperscan go
```

Comment:
change to make variable instead of env: `make hyperscan go MXE_PLUGIN_DIRS=plugins/go`